data/css/_colors.scss

```diff
@@ -1,3 +1,5 @@
+$brand-color: #005a71 !default;
```

Comment:
you'll notice they specify three shades of accent color (we only offer two) and an additional brand-color which we don't have.